samza-log4j/src/main/java/org/apache/samza/logging/log4j/StreamAppender.java

```diff
@@ -408,4 +393,15 @@ private void setSerde(Log4jSystemConfig log4jSystemConfig, String systemName, St
   public Serde<LoggingEvent> getSerde() {
     return serde;
   }
+
+  /**
+   * If the partition count was explicitly specified, then use that. Otherwise, use the container count as the partition
+   * count.
+   */
+  private int calculateStreamPartitionCount(Config config) {
```

Comment:
nit: since getPartitionCount is restored, we dont need this right

Reply:
Nice catch!